back/boxtribute_server/business_logic/warehouse/product/crud.py

```diff
@@ -171,25 +171,43 @@ def enable_standard_product(
     if price < 0:
         return InvalidPrice(value=price)
 
-    if (
-        product_id := Product.get_or_none(
+    # Find previous instantiations
+    previous_instantiations = (
```

Comment:
this should be max. 1 maybe add silent notfication to sentry if there are more than one